e2e/cypress/integration/account_settings/general/fullname_spec.js

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+import {getRandomInt} from '../../../utils';
+
+describe('Account Settings > Sidebar > General', () => {
+    // # number to identify particular user
+    const uniqueNumber = getRandomInt(1000);
+    before(() => {
+        // # Go to Account Settings as new user
+        cy.loginAsNewUser().as('newuser');
+        cy.toAccountSettingsModal(null, true);
+
+        // # Click General button
+        cy.get('#generalButton').click();
+
+        // # Open Full Name section
+        cy.get('#nameDesc').click();
+
+        // * Set first name value
+        cy.get('#firstName').clear().type(`정트리나${uniqueNumber}/trina.jung/집단사무국(CO)`);
+
+        // # save form
+        cy.get('#saveSetting').click();
```

Comment:
Currently the testcase fails because the new user might not be part of the team 'ad-1'. Hence when we search for that user, it would return users in the same team and channel and it will fail. In order to overcome this issue, you can first login as user-2, then visit the town-square channel, get the teamId and then use the same team id with loginAsNewUser function. Something like this:

```
        cy.apiLogin('user-2');
        cy.visit('/ad-1/channels/town-square');
        
        cy.getCurrentTeamId().then((teamId) => {
            cy.loginAsNewUser({}, [teamId]).as('newuser');

            // # Go to Account Settings as new user
            cy.toAccountSettingsModal(null, true);

            // # Click General button
            cy.get('#generalButton').click();

            // # Open Full Name section
            cy.get('#nameDesc').click();

            // * Set first name value
            cy.get('#firstName').clear().type(`정트리나${uniqueNumber}/trina.jung/집단사무국(CO)`);

            // # save form
            cy.get('#saveSetting').click();
        });
```